app/components/concept/index.ts

```diff
@@ -22,7 +22,7 @@ interface Signature {
   Element: HTMLDivElement;
 }
 
-interface BlockGroup {
+export interface BlockGroup {
```

Comment:
Let's move this out into `models/concept.ts` - so that models don't need to import from the view layer